play/PlayerRules/Basic Rules.txt

```diff
@@ -131,15 +131,27 @@ Feats		- Your character will gain special perks as you level up. This
 Player Stats:
 ==============================
 
-Each player has what are called Base Stats. An average person has a '5'
-for each stat, but an average player has a '6' . These stats are:
+Each player has what are called Base Stats. An average person has a '5' for
+each stat, but an average player has a '6'. Base Stats are split into Primary
+and Secondary stats. Primary stats are determined by your Base Stat points.
+Secondary stats are derived from several primary stats. For every point in a primary
+stat, a secondary stat point is awarded (if the primary and secondary stats are
+associated). The maximum number of points that can be put into a given secondary stat
+is determined by the quantity of points in the largest associated primary stat.
+These stats are as follows (secondary stat associations are indented):

```

Comment:
> " The maximum number of points that can be put into a given secondary stat is determined by the quantity of points in the largest associated primary stat."

Means that if I have 8 points in STR and 10 points in DEX, I can put up to 10 points into movement and up to 8 points into Carry Ability. How can i reword this to be more clear?

Also, should I leave "Carry Ability" as it currently is spelled? All carry ability does is negate max-movement penalties from carrying heavy armor or large weapons. I just want it to be a term that is as clear as possible.